scripts/bootstrap.sh.in

```diff
@@ -57,6 +57,7 @@ SALT_CALL=${SALT_CALL:-salt-call}
 
 declare -a PACKAGES=(
     salt-minion-@@SALT_VERSION
+    python3-psutil
```

Comment:
No, it's just today in bootstrap script we do not have any "override name" for packages we install, and btw `python3-psutil` work for both RHEL7 and RHEL8. It will install `python36-psutil` on RHEL7 and `python3-psutil` on RHEL8.

I could have use some more rendering (same as what we do for `@@VERSION`, `@@SALT_VERSION`) just did not want to add more complexity here just for a `psutil` package that will be installed anyway by salt-minion  But I could if it's confusing

Reply:
I added a comment I think it's good enough for this specific case we do not need complex rendering stuff

Reply:
Agreed. I'm thinking it would make sense, once we rework our scripts to be part of a single Python-based CLI, to get this list of packages directly from `versions.py`, and filter based on OS at runtime.